packages/kbn-search-api-panels/components/ingest_pipeline_panel.tsx

```diff
@@ -0,0 +1,166 @@
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useState, Fragment, useMemo } from 'react';
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+  EuiBadge,
+  EuiSuperSelect,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { IngestGetPipelineResponse, IngestPipeline } from '@elastic/elasticsearch/lib/api/types';
+
+import { DEFAULT_INGESTION_PIPELINE } from '../constants';
+
+interface OptionItem {
+  value: string;
+  inputDisplay: string;
+  dropdownDisplay: JSX.Element;
+}
+
+interface IngestPipelinePanelProps {
+  setSelectedPipeline: (pipeline: string) => void;
+  ingestPipelinesData?: IngestGetPipelineResponse;
+}
+
+interface IngestPipelineWithDeprecated extends IngestPipeline {
+  deprecated?: boolean;
+}
+
+export const IngestPipelinePanel: React.FC<IngestPipelinePanelProps> = ({
+  setSelectedPipeline,
+  ingestPipelinesData,
+}) => {
+  const createProcessorCount = (item: IngestPipelineWithDeprecated | undefined) => (
+    <EuiFlexItem grow={false}>
+      <EuiText size="s" color="subdued">
+        <p>
+          {i18n.translate('searchApiPanels.welcomeBanner.ingestPipelinePanel.processorCount', {
+            defaultMessage: '{count} {count, plural, one {processor} other {processors}}',
+            values: { count: item?.processors?.length },
+          })}
+        </p>
+      </EuiText>
+    </EuiFlexItem>
+  );
+
+  const createManagedBadge = (item: IngestPipelineWithDeprecated | undefined) =>
+    item?._meta?.managed && (
+      <EuiFlexItem grow={false}>
+        <EuiBadge>
+          {i18n.translate('searchApiPanels.welcomeBanner.ingestPipelinePanel.managedBadge', {
+            defaultMessage: 'Managed',
+          })}
+        </EuiBadge>
+      </EuiFlexItem>
+    );
+
+  const createRecommendedBadge = (pipelineName: string) =>
+    pipelineName === DEFAULT_INGESTION_PIPELINE && (
+      <EuiFlexItem grow={false}>
+        <EuiBadge color="primary">
+          {i18n.translate('searchApiPanels.welcomeBanner.ingestPipelinePanel.recommendedBadge', {
+            defaultMessage: 'Recommended',
+          })}
+        </EuiBadge>
+      </EuiFlexItem>
+    );
+
+  const options = useMemo(() => {
+    const createOptionItem = (pipelineName: string): OptionItem => {
+      const item: IngestPipelineWithDeprecated | undefined = ingestPipelinesData?.[pipelineName];
+      return {
+        value: pipelineName,
+        inputDisplay: pipelineName,
+        dropdownDisplay: (
+          <Fragment>
+            <strong>{pipelineName}</strong>
+            <EuiFlexGroup
+              gutterSize="s"
+              alignItems="center"
+              data-test-subj="ingestPipelinePanelProcessors"
+            >
+              {createProcessorCount(item)}
+              {createManagedBadge(item)}
+              {createRecommendedBadge(pipelineName)}
```

Comment:
I think, we want to show recommended ingestion pipeline as first item ?
<img width="1326" alt="Screenshot 2024-05-09 at 10 40 51 AM" src="https://github.com/elastic/kibana/assets/55930906/3d764146-3302-4849-b911-f1084cec2130">

Design: 
<img width="831" alt="Screenshot 2024-05-09 at 10 41 36 AM" src="https://github.com/elastic/kibana/assets/55930906/821e55d2-38cc-4552-9da1-a0f9ff2d161a">

Reply:
I think you are right. Let me make this change.

Reply:
Updated